news/2022-01-28-v2.0.0.0a13.md

```diff
@@ -0,0 +1,30 @@
+# v2.0.0a13
+
+Starting wits this release Resoto ships as multiple independent Docker containers. See [packages](https://github.com/orgs/someengineering/packages?repo_name=resoto) for a list of available containers. The single container, which includes all dependencies still exist.
```

Comment:
We should not link to the GitHub package list, since we want users to consume our packages via the Scarf Gateway.

Also, "with" is misspelled.